app/controllers/application_controller.rb

```diff
@@ -44,6 +48,18 @@ def infer_params_key(resource)
     klass.name.underscore.to_sym
   end
 
+# Sorting
+
+  # Sort by :id by default, so there is a consistent order even when no order has been
+  # selected via the interface
+  def default_sort_options
+    {id: :desc}
+  end
+
+  def sort(collection)
+    ModelSorter.sort(collection, params, default_sort_options)
```

Comment:
Hey @osiro in that case is there any reason you wouldn't just override the sort method in the subclass?

```ruby
def sort(collection)
  MyCustomSorter.sort(collection, params, default_sort_options)
end
```